src/wp-admin/includes/image-edit.php

```diff
@@ -43,15 +43,16 @@ function wp_image_editor( $post_id, $msg = false ) {
 			$note = "<div class='notice notice-success' tabindex='-1' role='alert'><p>$msg->msg</p></div>";
 		}
 	}
-	$edit_custom_sizes = false;
+
 	/**
-	 * Filters whether custom sizes are available options for image editing.
+	 * Shows the settings in the Image Editor that allow selecting to edit only the thumbnail of an image.
 	 *
-	 * @since 6.0.0
+	 * @since 6.3.0
 	 *
-	 * @param bool|string[] $edit_custom_sizes True if custom sizes can be edited or array of custom size names.
+	 * @param boolean Whether to show the settings in the Image Editor. Default false.
```

Comment:
```suggestion
	 * @param bool Whether to show the settings in the Image Editor. Default false.
```
Nit: Core uses `bool`